### ISSUE TITLE
Default link text color does not meet A11y contrast ratio guidelines against white background

### ISSUE THREAD

**Issue by m7kvqbe1** [Royal-Navy/design-system#3961]
Change to action 600 instead to meet guidelines WCAG AA.
